website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,71 +1,87 @@
 
-!!! Info
+## Component availability
 
-These best practices have been curated by the Design Systems Team, Figma has its own set of [best practices](https://www.figma.com/best-practices/) for using the tool effectively.
+**Structure** is the legacy set of components and foundations used to connect HashiCorp products and consists Ember components and foundational styles paired with a limited set of Figma counterparts. **Structure is no longer supported and is in the process of deprecation and sunsetting.**
 
-!!!
+As we work to achieve parity between Structure and Helios, use this decision tree to understand how to move forward within your project:
 
-**Note:** Much of this already exists from Design Ops, waiting to collaborate with them on the content.
+![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
 
-## File hygiene
+**Does the element (or a comparable alternative) exist in Helios?**
 
-- Use a cover photo
-- Differentiate between delivered work, and work in progress
+If so, great! Using this element ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
 
-### Naming conventions
+**If no, does the element (or a comparable alternative) exist in Structure?**
 
-Naming things is hard, and while naming components and elements within a design system is often different from naming elements further down the UX funnel, the same best practices can be applied.
+If yes, use this component with the understanding that:
 
-!!! Insight
+1. Structure will be deprecated/sunsetted in the future
+2. Not all Figma components are built in code
+3. There is no longer a support structure in place if errors/bugs arise
 
-Whatever method you choose to use, the most important thing is maintaining a consistent naming strategy across projects, files, and components.
+If no, or having future support is a must: design and build your own local element using Helios foundations (color, typography, elevation, etc) as a starting point.
+
+### Local components
+
+Components and patterns that are unique to your product should be designed and built locally, extending Helios components and foundations.
+
+![Local component patterns](/assets/getting-started/designers/local-component-patterns.png)
+
+<LinkTo class="doc-link-generic" @route="patterns">Patterns</LinkTo> and guidance around combining and extending Helios components are not currently supported, but are on our [roadmap](https://go.hashi.co/hds-rollout).
```

Comment:
I'd like to see us try to use different language when talking about patterns because I don't think "not supported" is entirely accurate. "Not supported" makes it sound like we made a conscious decision not to do something, but really we just needed to focus on lower-level components so that we could build out these larger patterns. I'm also not sure how "guidance around combining and extending Helios components" is something that can or cannot be supported.

Reply:
I'm not sure I agree with this because there's some additional context about what items are on the roadmap. Though, I'll try to clean it up a little bit to lean on our future plans for patterns, we just don't have them yet (something like "provide", etc).